align/compiler/create_array_hierarchy.py

```diff
@@ -44,10 +44,9 @@ def __init__(self, ckt, match_pairs):
                 isinstance(const, constraint.GroundPorts) or \
                 isinstance(const, constraint.ClockPorts):
                 self.stop_points.extend(const.ports)
-            elif isinstance(const, constraint.IdentifyArray):
-                self.condition = const.isTrue
-            elif isinstance(const, constraint.IsDigital):
-                self.is_digital = const.isTrue
+            elif isinstance(const, constraint.CompilerOpt):
+                self.condition = getattr(const, 'identify_array', False)
```

Comment:
Instead of spreading defaults around the code, it makes it tidier to set default value in the schema.

Reply:
done